clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -698,6 +699,16 @@ scheduler_tmpl_t<RecordType, ReaderType>::init(
                 tid2input_[workload_tid_t(workload_idx, it.first)] = it.second;
             }
         }
+        if (!workload.times_of_interest.empty()) {
+            for (const auto &modifiers : workload.thread_modifiers) {
+                if (!modifiers.regions_of_interest.empty())
```

Comment:
Meaning one cannot specify regions and times of interest both?

Reply:
Right.  Added a comment.